src/SmartComponents/Inventory/applications/vulnerabilities/DataMapper.js

```diff
@@ -18,13 +10,22 @@ export function createCveListBySystem({ isLoading, ...rest }) {
             data: data.map(row => ({
                 id: row.id,
                 cells: [
-                    <span key={ row.id }>
-                        <SecurityIcon size="md" color={ colorToImpact[row.attributes.impact] } />
-                    </span>,
+                    <Shield
+                        impact={ row.attributes.impact }
+                        hasTooltip={ true }
+                        tooltipPosition={ 'right' }
+                        key={ row.id.toString() }
+                    />,
                     row.attributes.synopsis,
-                    <span key={ `title-${row.id}` } title={ row.attributes.description }>{ row.attributes.description.substr(0, 199) }...</span>,
+                    <LongTextTooltip
+                        content={ row.attributes.description }
+                        maxLength={ 200 }
+                        tooltipPosition={ 'top' }
```

Comment:
Can we make sure it's consistent with https://github.com/RedHatInsights/vulnerability-ui/blob/master/src/Helpers/VulnerabilitiesHelper.js#L19-L24